static/src/stylesheets/_vars.scss

```diff
@@ -114,6 +114,10 @@ $frontend-palette: (
     crosswordAccent1:       guss-colour(multimedia-support-1, $pasteup-palette),
     crosswordAccent2:       guss-colour(news-main-2, $pasteup-palette),
 
+    sudoku-accent-1:        guss-colour(multimedia-support-1, $pasteup-palette),
```

Comment:
Why don't these follow the same naming conventions of the variables above?

e.g. `x-default`, `x-accent` and `x-mute`